dropbox.in

```diff
@@ -663,19 +663,21 @@ class DropboxCommand(object):
 
         self.f.flush()
 
-        # Start a ticker
-        ticker_thread = CommandTicker()
-        ticker_thread.start()
+        if sys.stderr.isatty():
+            # Start a ticker
+            ticker_thread = CommandTicker()
+            ticker_thread.start()
```

Comment:
small request to make it a little less bugprone w/ changes in the future

```
ticker_thread = None
if sys.stderr.isatty():
    ticker_thread = CommandTicker()
    ticker_thread.start()
```

and

```
if ticker_thread is not None:
    ticker_thread.stop()
    ticker_thread.join()
```

It's helpful so that any changes to the condition in the future don't have risk of a bug where it's not propagated. The full state is encoded into the ticker_thread() object directly. Thanks!